llvm/test/CodeGen/AMDGPU/llvm.amdgcn.image.sample.noret.ll

```diff
@@ -0,0 +1,479 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc -mtriple=amdgcn -mcpu=gfx1030 -verify-machineinstrs -global-isel=0 < %s | FileCheck -check-prefixes=GFX10PLUS,GFX10PLUS-SDAG,GFX10,GFX10-SDAG %s
```

Comment:
Drop all the -verify-machineinstrs, and move the -global-isel arguments to be first

Reply:
Done